ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py

```diff
@@ -32,8 +32,11 @@ def _get_range_for_grid(draw):
 
 
 @st.composite
-def _get_dtype_and_range(draw):
-    dim = draw(helpers.ints(min_value=2, max_value=5))
+def _get_dtype_and_range(draw, dimension=None):
+    if dimension!=None:
+        dim = dimension
+    else:
+        dim = draw(helpers.ints(min_value=2, max_value=5))
```

Comment:
After the changes I mentioned this will be unecessary as the function should accept mutlidimensional inputs for start and stop